charts/ti-key-release-1/templates/configmap/tsi-mutate-configmap.yaml

```diff
@@ -5,6 +5,35 @@ metadata:
   name: tsi-mutate-webhook-configmap
 data:
   tsiMutateConfig.yaml: |
+      initContainers:
+      - name: jwt-init
+        image: {{ .Values.tiRepo }}/tsi-jwt-init:{{ .Chart.AppVersion  }}
+        imagePullPolicy: Always
+        securityContext:
+          runAsUser: 0
+        env:
+        - name: HOST_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.hostIP
+        - name: JWT_TTL_SEC
+          value: {{ .Values.jwt.expireSec }}
+        - name: VAULT_ADDR
+          value: {{ .Values.vaultAddress }}
+        - name: SECRET_REFRESH_SEC
+          value: {{ .Values.secrets.refreshSec }}
```

Comment:
Note to self - we probably need to move all these into a config map for operator conversion process.